src/sentry/api/endpoints/group_tags.py

```diff
@@ -24,17 +24,8 @@ def get(self, request, group):
         else:
             value_limit = 10
 
-        use_snuba = request.GET.get('enable_snuba') == '1'
```

Comment:
Did you verify this is never false? It seems here we are not relying on snuba.events-queries.enabled.

Reply:
Yea, it's only called from the UI in one place here https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/group/sidebar.jsx#L83 and I've removed this line in this PR

Reply:
what about external API calls? this is the public api. 
I would advise you verified if this is happening with james. the query string is probably in the nginx log that is now stored in clickhouse.

Reply:
I'm going to move this into a separate PR since it's changing behavior for external API calls made without the `enable_snuba` flag.